openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/db2Connection.json

```diff
@@ -48,9 +48,9 @@
       "description": "Host and port of the DB2 service.",
       "type": "string"
     },
-    "databaseSchema": {
-      "title": "databaseSchema",
-      "description": "databaseSchema of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single databaseSchema. When left blank, OpenMetadata Ingestion attempts to scan all the databaseSchema.",
+    "database": {
+      "title": "database",
+      "description": "Database of the data source.",
```

Comment:
any particular reason we changed the databaseSchema to database?

Reply:
I think both will work similarly,
based on the code in source_connections

         if hasattr(connection, "database"):
                url += f"/{connection.database}" if connection.database else ""
        
            elif hasattr(connection, "databaseSchema"):
                url += f"/{connection.databaseSchema}" if connection.databaseSchema else ""

Reply:
They will work similarly to create the connection but not to perform the ingestion. If we leave "databaseSchema", the database name will be `default,` and we want to start ingestion from the database name, not from a schema.